docs/deployment-github-pages.md

```diff
@@ -26,24 +26,20 @@ Navigate to your repository settings, click on Pages and enable GitHub pages. Wh
 
 To trigger action, push your code with the workflow to main.
 
-:::{warning} Custom Domains
-GitHub allows you to host your static content on a custom domain, doing so _may_ require you to change the `BASE_URL` environment variable in the action. If you have unstyled content, try changing the `BASE_URL` to a blank string: `BASE_URL=''` (note the **single quotes**!); this serves the build assets from the root of your domain, rather than the default, which is the name or your repository.
-:::
-
-:::{tip} `BASE_URL` environment variable
-:class: dropdown
-The build and site assets are in the `/build` folder, which would point outside of the current repository to a repository called 'build', which probably doesn't exist!
+:::{tip} You will probably need to set a `BASE_URL`
+GitHub Pages likely requires you to set a `BASE_URL` because the URL for GitHub Pages defaults to a sub-folder of `username.github.io` (e.g. `username.github.io/myrepository/`).
+The easieset thing to do is use the GitHub Action variable for the repository name:
 
-To fix this, we can change the base url that the site is mounted to, which can be done through the `BASE_URL` environment variable:
-
-```bash
-export BASE_URL="/repository_name"
+```yaml
+BASE_URL: /${{ github.event.repository.name }}
 ```
 
-The base URL is _absolute_ and should not end with a trailing slash. This can be done automatically in a GitHub Action by looking to the `github.event.repository.name` variable.
+If you're using a custom domain (e.g. `mydomain.org`) then you may need to manually set `BASE_URL` to an empty string, like `export BASE_URL=''`.
```

Comment:
Just a note that i did not need to do this (i think)

see [here](https://github.com/pyOpenSci/peer-review-metrics/blob/main/.github/workflows/deploy.yml#L12)

now i may not understand how this works yet but i didn't have to tell it my domain name, i did have to define the subfolder / repo name